### DISCUSSION TITLE
Which composer wrote the opera cycle Der Ring des Nibelungen cycle?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#153] [Galaxy Trivia] [answered]
**Hint:** Richard

**Comment by YuetianW** [comment, marked as answer]
The composer who wrote Der Ring des Nibelungen cycle was [**Richard Wagner**](https://www.britannica.com/topic/Der-Ring-des-Nibelungen).He also wrote the German librettos for the four music dramas
